src/clj/webchange/assets/core.clj

```diff
@@ -0,0 +1,68 @@
+(ns webchange.assets.core
+  (:require [webchange.db.core :refer [*db*] :as db]
+            [clojure.java.io :as io]
+            [clojure.string :refer [join]]))
+
+(import 'java.security.MessageDigest
+        'java.math.BigInteger)
+
+(defn directory
+  [config]
+  (let [dir (or (:dataset-dir config) "resources/public/raw/")]
```

Comment:
This probably should be :public-dir?

there would be 2 folders to reindex hashes.
public/raw
public/upload